lib/sentry/integrations/check_in_id_mappings.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Sentry.Integrations.CheckInIDMappings do
+  @moduledoc false
+
+  use GenServer
+
+  @table :cron
```

Comment:
Just a nit to avoid a very-common name that could be in use by users or other applications.

```suggestion
  @table :sentry_cron_mappings
```